package.json

```diff
@@ -1,25 +1,24 @@
 {
   "name": "peviitor_jsscraper",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
This version is from 20 days ago

<img width="1440" alt="Screenshot 2023-08-18 at 22 36 16" src="https://github.com/peviitor-ro/peviitor_jsscraper/assets/67306273/de08d288-ee9a-448d-8bf9-a17f098a6dcf">